tests/console/validate_modify_existing_partition.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify the partition modified in modify_existing_partition.
+# Maintainer: Jonathan Rivrain <jrivrain@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data  = get_test_data();
+    my $fs_type    = $test_data->{fs_type};
+    my $partition  = $test_data->{existing_partition};
+    my $mountpoint = $test_data->{mount_point};
+    my $part_size  = $test_data->{part_size};
+    $part_size =~ s/[A-Za-z]//g;
```

Comment:
That's a bit tricky, if I will write 1024MiB in test data, we will still use GB for comparison.
So I would either put unit in format it's in the test, or put extra setting for size units.

Reply:
Right, I should have thought of it. This is stripping out the unit actually. lsblk displays the partition size in human-readable format by default, in *ibibytes. It will be tricky to handle all the possible units... I'd rather make it mandatory to use MiB for example.

Reply:
to write GiB explicitly does not make it dynamic to reuse this test module in the future.

Reply:
I would not force anything, even if it makes sense, or spend time doing operation over result, plus human readable is still good for the tester checking the test, it is better and cleaner to set your expectation in the yaml.

Reply:
Applied solution disscussed with @jknphy , add a variable in the yaml and put a comment there. Cheap but simple.